wa/workloads/antutu/__init__.py

```diff
@@ -90,12 +114,24 @@ def __init__(self, device, **kwargs):
         super(Antutu, self).__init__(device, **kwargs)
         self.gui.timeout = 1200
 
+    def initialize(self, context):
+        super(Antutu, self).initialize(context)
+        #Install the supporting benchmark
+        supporting_apk = context.get_resource(ApkFile(self, package='com.antutu.benchmark.full'))
+        self.target.install(supporting_apk)
```

Comment:
Do we have `supporting_apks` for previous versions or should this only be if the version is == 10.X?

Reply:
`supporting_apks` have been required since at least version 8 which was released in January 2020.

Reply:
Ok, stupid question but in that case how have we been handling the `supporting_apk` before this change? 

Also for backwards compatibility with v7 should we add a >=8 check?

Reply:
I don't believe the public workload has. I have been running it in a forked repo since v8. 

We could add a check but V7 is so old and almost impossible to get hold of these days. Happy to add if you want though.